prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -172,8 +182,8 @@ public async Task SendSiteRegistrationAsync(Site site)
                 ("SiteRegistrationReview.pdf", await _razorConverterService.RenderViewToStringAsync("/Views/SiteRegistrationReview.cshtml", site)),
                 ("BusinessLicence.pdf", await _razorConverterService.RenderViewToStringAsync(businessLicenceTemplate, businessLicenceDoc))
             }
-            .Select(content => (Filename: content.Filename, Content: _pdfService.Generate(content.HtmlContent)))
-            .Select(pdf => new Attachment(new MemoryStream(pdf.Content), pdf.Filename, "application/pdf"));
+            .Select(content => (Filename: content.Filename, Content: _pdfService.Generate(content.HtmlContent)));
+            // .Select(pdf => new Attachment(new MemoryStream(pdf.Content), pdf.Filename, "application/pdf"));
```

Comment:
What is the reason for the commented out line?

Reply:
Is it something that we need later or something that isn't really needed? As it looks like all the functions changed for what they accept as attachments so could potentially just remove this line.